.prettierrc.json

```diff
@@ -1,10 +1 @@
-{
-  "arrowParens": "always",
-  "bracketSpacing": true,
-  "semi": true,
-  "trailingComma": "all",
-  "singleQuote": true,
-  "printWidth": 120,
```

Comment:
A lot of the churn in these files is moving from 180 to 120 width. Would it be worth keeping 180 to reduce the churn?

Reply:
I'm fine with whatever, as long as it's consistent across the services. Though I suspect moving to 180 will cause churn in the services that had already been at 120... I'll check

Reply:
As I suspected, going 120 -> 180 just churns a different set of files. Actually, 62 files vs. 61 the other way. 🤷🏻